src/lib/iconvenc/iconvenc.pas

```diff
@@ -0,0 +1,224 @@
+{
+    This file is part of the Free Pascal run time library.
+    Copyright (c) 2000 by Marco van de Voort(marco@freepascal.org)
+    member of the Free Pascal development team
+
+    libiconv header translation + a helper routine
+    http://wiki.freepascal.org/iconvenc
+
+    See the file COPYING.FPC, included in this distribution,
+    for details about the copyright. (LGPL)
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+}
+unit iconvenc;
+
+interface
+{$mode objfpc}{$H+}
+
+
+uses
+  ctypes,
+  {$IFNDEF MSWINDOWS}
+    unixtype,
+    baseunix,
+  {$ENDIF}
+  initc;
+
+{$ifdef beos}
+  const
+    ESysEILSEQ = EILSEQ;
+{$endif}
+
+
+{$IFDEF MSWINDOWS}
+  type
+    iconv_t = Pointer;
+
+  const
+    iconvlib = 'libiconv.dll';
```

Comment:
Any reason why you didn't use libiconv-2.dll?

Reply:
This is the `libiconv-2.dll` that I renamed for no good reason. Renamed it back.